package.json

```
@@ -9,10 +9,13 @@
     "src"
   ],
   "scripts": {
+    "prepublishOnly": "rollup -c",
     "eslint": "eslint src",
     "eslint-fix": "npm run eslint -- --fix",
-    "prepublishOnly": "rollup -c",
-    "test": "npm run test-coverage && npm run eslint",
+    "prepack": "npm run compile",
```

this file seems wrong because compile is not defined.. I would also try to keep the name of the script in alphabetical order.

```suggestion
        "prepack": "rollup -c",
```